src/framework/components/gsplat/component.js

```diff
@@ -469,7 +500,7 @@ class GSplatComponent extends Component {
         // create new instance
         const asset = this._assetReference.asset;
         if (asset) {
-            this.instance = new GSplatInstance(asset.resource, this._materialTmp);
+            this.instance = new GSplatInstance(asset.resource, this._materialTmp, this._fullSH);
```

Comment:
I'd stick `this._fullSH` to options class, as this is optional and would be easier to add additional properties later